docs/src/site/_config.yml

```diff
@@ -0,0 +1,5 @@
+name: http4s.org
+markdown: redcarpet
+highlighter: pygments
```

Comment:
Huh. I suggested this because without it I couldn't run `bin/local-site` without it. Fun.